CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
```

Comment:
It isn't quite on par with https://github.com/giantswarm/k8scloudconfig/pull/818/files

E.g. `DockerhubToken` validation is missing. Isn't this cherry-pick'able?